src/index.js

```diff
@@ -183,16 +183,20 @@ export default class SplunkEvents {
     }
     let screen = window.screen ? window.screen : {}
     let location = window.location ? window.location : {}
-    return (
-      `additional_info="${navigator.userAgent.replace(/\,/g, ';')},` +
+
+    const additionaInfo =
+      `${navigator.userAgent.replace(/\,/g, ';')},` +
       `${navigator.browserLanguage || navigator.language},` +
       `${navigator.platform},${screen.availWidth || '-'},${
         screen.availHeight || '-'
       },${location.hostname},` +
       `${location.pathname},${location.protocol.replace(':', '')},${
         location.hash || '-'
-      }"`
-    )
+      }`
+
+    return {
+      additional_info: additionaInfo,
```

Comment:
Isn't more JavaScriptesque to use came case for object keys? We even use came case for widely-used Splunk fields, such as `workflowType` and `workflowInstance`.

Reply:
Agreed but that would be a breaking change for someone that it was using  it before....

Reply:
Ok. So I'd only ask to correct the variable name, which is mistyped `additionaInfo`